### PR TITLE
func returns None, updated test and doc

### PR DESCRIPTION
**Pull request by shubham-shay-te**

### CODE REVIEW

File: core_lib/helpers/func_utils.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def get_func_parameters_as_dict(func, *args, **kwargs) -> dict:
         elif param in parameters and parameters[param].default != inspect.Parameter.empty:
```
Comment (shay-te) [inline core_lib/helpers/func_utils.py:46]: question please. 
how dose this effect `build_value_by_func_parameters` and it's tests

File: website/docs/function_utils.md [hunk 2]
```diff
@@ -34,9 +34,9 @@ formatted_parameters = build_value_by_func_parameters('key_{param_1}_{param_2}_{
 print(formatted_parameters) # key_1_2_hello world
 
 formatted_parameters = build_value_by_func_parameters('key_{param_1}_{param_2}_{param_3}', function_to_format, 1) 
-print(formatted_parameters) # key_1_param_2_hello
+print(formatted_parameters) # key_1_None_hello
```
Comment (shay-te) [inline website/docs/function_utils.md:37]: !E E! ? instead of None?

**Review by shay-te — CHANGES_REQUESTED** [review]

File: website/docs/function_utils.md [hunk 3]
```diff
@@ -34,9 +34,9 @@ formatted_parameters = build_value_by_func_parameters('key_{param_1}_{param_2}_{
 print(formatted_parameters) # key_1_2_hello world
 
 formatted_parameters = build_value_by_func_parameters('key_{param_1}_{param_2}_{param_3}', function_to_format, 1)
```
Comment (shay-te) [inline website/docs/function_utils.md:36]: who is `function_to_format` ?what parameteres dose it recieved. do it have default values?

**Review by shay-te — COMMENTED** [review]
gppd work Shubham thank you

File: website/docs/function_utils.md (hunk 3 above)
Comment (shubham-shay-te) [reply website/docs/function_utils.md:36]: the function whose params will be formatted, it is on line 30, it has both types of parameters

**Review by shay-te — APPROVED** [review]
Thank you Shubham. good work !!

**Pull request merged**